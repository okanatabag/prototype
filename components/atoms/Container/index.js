import styled from 'styled-components';

const Container = styled.div`
  display: block;
  position: relative;
  background: ${(props) => props.bg || '#ffffff'};
`;

export default Container;
