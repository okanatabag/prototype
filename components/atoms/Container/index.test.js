import React from 'react';
import renderer from 'react-test-renderer';
import Container from '.'
it('renders correctly', () => {
    const tree = renderer
      .create(<Container bg="#000000">Test</Container>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});