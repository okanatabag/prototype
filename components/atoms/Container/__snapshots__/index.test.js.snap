// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Container-sc-7r4j6o-0 bwzCzR"
>
  Test
</div>
`;
